postcss.mailer.config.js

```diff
@@ -7,9 +7,9 @@ module.exports = {
   plugins: [
     require('postcss-import')(postcssImportConfig),
     require('postcss-extend-rule'),
-    require('postcss-nested'),
-    require('tailwindcss'),
     require('postcss-css-variables'),
```

Comment:
@jagthedrummer according to [this caniemail.com report, css variables have bad support among email clients](https://www.caniemail.com/search/?s=variables).

So yeah, let's only keep postcss-css-variables for the mailer css, and add a comment about this caniemail.com report for posterity.

Otherwise, LGTM

Reply:
Great, idea about linking to that report, @pascallaliberte. Thanks!